.travis.yml

```diff
@@ -1,14 +1,12 @@
-sudo: required
 language: generic
-dist: trusty
+services:
+  - docker
 
-matrix:
-  include:
-  - name: "Kinetic"
```

Comment:
No more Kinetic? HERO runs kinetic AFAIK

Reply:
Damn ur fast, I was just checking whether this works under melodic (18.04). Since pip apparently seems to have problems on 16.04.

Reply:
also not working :(

Reply:
> Damn ur fast

Tuesday evening...